bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseThingHandler.java

```diff
@@ -78,15 +94,202 @@ public void initialize() {
         updateStatus(ThingStatus.UNKNOWN);
     }
 
+    @Override
+    public void onDeviceAdded(AVMFritzBaseModel device) {
+        // nothing to do
```

Comment:
If onDeviceRemove/onDeviceGone cause the thing to go offline, wouldn't adding the device make the thing go online?

Reply:
No, `onDeviceAdded()` will only be called if no Thing is available.